src/components/common/EntityInfo.vue

```diff
@@ -44,26 +44,25 @@ import { Getter } from 'vuex-class'
 
 // Interfaces
 import { GetterIF } from '@/interfaces'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
 
 @Component({})
 export default class EntityInfo extends Vue {
   // Global getters
   @Getter isEntityType!: GetterIF
-  @Getter isTypeBcomp!: GetterIF
-  @Getter isTypeCoop!: GetterIF
+  @Getter getEntityType!: CorpTypeCd
   @Getter getNameRequestNumber!: GetterIF
   @Getter getTempId!: GetterIF
   @Getter getApprovedName!: GetterIF
 
-  /** The entity title  */
-  private entityTitle (): string {
-    if (this.isTypeBcomp) {
-      return 'BC Benefit Company Incorporation Application'
-    } else if (this.isTypeCoop) {
-      return 'Incorporate a BC Cooperative Association'
-    }
+  /** The entity application title  */
+  private get entityTitle (): string {
+    return `${GetCorpFullDescription(this.getEntityType)} Incorporation Application`
+  }
```

Comment:
I'd prefer to see something like this:
`${this.getCorpTypeDescription(type)} ${FilingNames.INCORPORATION_APPLICATION}`

but you'll have to create a new FilingNames enum:
https://github.com/bcgov/business-filings-ui/blob/main/src/enums/filingNames.ts